frontend/src/pages/CourseEditor/LessonsStep/LessonItem/styled.ts

```diff
@@ -0,0 +1,84 @@
+import { styled, Box, Input } from '@mui/material';
+
+export const LessonItemWrapper = styled(Box)({
+  margin: '32px 0 50px 0',
+});
+
```

Comment:
wrong UI
<img width="705" alt="image" src="https://user-images.githubusercontent.com/44285339/168602847-ca69a76b-210e-474c-86dc-ae8e20ea21c2.png">
<img width="608" alt="image" src="https://user-images.githubusercontent.com/44285339/168602892-bc77fdaf-7c1a-473a-a24e-569a360fe630.png">

Reply:
also in case of removing 2nd lesson, the # of next lesson should be 2 not 3

Reply:
add more lessons button is not working, blank page appears

Reply:
why all the fields have different font styles and size?

Reply:
not all the values changes in formik values, please fix it and provide proper structure